### DISCUSSION TITLE
Is it possible to put this script on ubuntu server?

### DISCUSSION THREAD

**Discussion by ForceDev1** [devlikeapro/waha#247] [Q&A]
I have my own software and want to use it as a generic api. Is it possible to put it on the server so that all my users can access the api?

**Comment by allburov** [comment]
Hi! Sure, run the same command as you'd do for personal use
https://waha.devlike.pro/

**Reply by ForceDev1** [reply]
I tried it, but the page won't load on a device that is outside the local network

**Reply by allburov** [reply]
It's more network related thing, has nothing to do with the project.
Probably you need to configure firewall right for that solution
